src/qt/overviewpage.cpp

```diff
@@ -197,7 +197,10 @@ void OverviewPage::setBalance(const interfaces::WalletBalances& balances)
     BitcoinUnit unit = walletModel->getOptionsModel()->getDisplayUnit();
     if (walletModel->wallet().isLegacy()) {
         if (walletModel->wallet().privateKeysDisabled()) {
+            ui->labelSpendable->setText(tr("Watch-only:"));
+            ui->labelSpendable->setVisible(true);
             ui->labelBalance->setText(BitcoinUnits::formatWithPrivacy(unit, balances.watch_only_balance, BitcoinUnits::SeparatorStyle::ALWAYS, m_privacy));
+            ui->labelBalance->setToolTip(tr("Your current balance in watch-only addresses"));
```

Comment:
Despite the confusing interfaces, watch-only wallets are not based on addresses (or they would never see spends). Prefer to use less confusing language here.

Reply:
Perhaps "Your current watch-only balance" or "Your current watch-only wallet balance"?